### ISSUE TITLE
Weakaura editor disappears after UI reload

### ISSUE THREAD

**Issue by nperickson** [WeakAuras/WeakAuras2#2809]
## Describe the bug

I was working on adjusting some weakauras and as I was editing one everything, including the weakauras editing screen, disappeared after a reload. /wa would no longer pull up and I had 2 errors in my bugsack (below)

**Do you have an error log of what happened?**

```plaintext
1x SharedXML\Backdrop.lua:251: TexCoord out of range
[string "=[C]"]: in function `SetTexCoord'
[string "@SharedXML\Backdrop.lua"]:251: in function <SharedXML\Backdrop.lua:250>
[string "@SharedXML\Backdrop.lua"]:283: in function `SetupTextureCoordinates'
[string "@SharedXML\Backdrop.lua"]:368: in function `ApplyBackdrop'
[string "@SharedXML\Backdrop.lua"]:384: in function `SetBackdrop'
[string "@WeakAuras\SubRegionTypes\Border.lua"]:54: in function `modify'
[string "@WeakAuras\RegionTypes\RegionPrototype.lua"]:611: in function `modifyFinish'
[string "@WeakAuras\RegionTypes\AuraBar.lua"]:1320: in function `modify'
[string "@WeakAuras\WeakAuras-3.1.4.lua"]:2859: in function `SetRegion'
[string "@WeakAuras\WeakAuras.lua"]:2754: in function <WeakAuras\WeakAuras.lua:2660>
[string "@WeakAuras\WeakAuras.lua"]:2789: in function `Add'
[string "@WeakAuras\WeakAuras.lua"]:2320: in function `load'
[string "@WeakAuras\WeakAuras.lua"]:2327: in function `AddMany'
[string "@WeakAuras\WeakAuras.lua"]:1009: in function <WeakAuras\WeakAuras.lua:981>

[string "@WeakAuras\WeakAuras-3.1.4.lua"]:1131: in function <WeakAuras\WeakAuras.lua:1082>
```
```plaintext
1x SharedXML\Backdrop.lua:251: TexCoord out of range
[string "=[C]"]: in function `SetTexCoord'
[string "@SharedXML\Backdrop.lua"]:251: in function <SharedXML\Backdrop.lua:250>
[string "@SharedXML\Backdrop.lua"]:283: in function `SetupTextureCoordinates'
[string "@SharedXML\Backdrop.lua"]:227: in function <SharedXML\Backdrop.lua:225>
[string "=[C]"]: in function `resume'
[string "@WeakAuras\WeakAuras-3.1.4.lua"]:1048: in function `Login'
[string "@WeakAuras\WeakAuras-3.1.4.lua"]:1131: in function <WeakAuras\WeakAuras.lua:1082>

Locals:
(*temporary) = <unnamed> {
 0 = <userdata>
}
(*temporary) = 0.257813
(*temporary) = 21747.968117
(*temporary) = 0.367188
(*temporary) = 21747.968117
(*temporary) = 0.257813
(*temporary) = 0.062500
(*temporary) = 0.367188
(*temporary) = 0.062500
```
### To Reproduce

1. Log into game
2. Enter world as character with bugsack, buggrabber, and weakauras enabled
3. Type /wa and the menu doesn't load, nor do you get any errors
4. Check bugsack

**Screenshots**

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**

**Which version of WeakAuras are you using?**

**3.1.4**

**Are you on World of Warcraft Classic or Retail?**

- [ ] Classic
- [X] Retail

**Was it working in a previous version? If yes, which was the last good one?**

[WeakAuras.zip](https://github.com/WeakAuras/WeakAuras2/files/5682669/WeakAuras.zip)

### Additional Info

[WeakAuras.zip](https://github.com/WeakAuras/WeakAuras2/files/5682670/WeakAuras.zip)

**Comment by nullKomplex** [comment]
> **Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**

Can confirm it happens with only WA enabled.

Additional Info: Group Scale of `Cupcakey Castbar` was set to 105. The slider goes from 0 to 2...

**Comment by nullKomplex** [comment]
This was not resolved by e4299c906cfd1ad2acd4e11eb1e90986e727cb7f

**Comment by nperickson** [comment]
Good catch, now that you mention it I was editing the Cupcakey castbar weakaura before the refresh and I must have set the scale incorrectly with a manual entry.
